Networking/Networking/Model/Product/Product.swift

```diff
@@ -380,7 +380,14 @@ public struct Product: Codable, GeneratedCopiable, Equatable, GeneratedFakeable
 
         let attributes = try container.decode([ProductAttribute].self, forKey: .attributes)
         let defaultAttributes = try container.decode([ProductDefaultAttribute].self, forKey: .defaultAttributes)
-        let variations = try container.decode([Int64].self, forKey: .variations)
+        let variations: [Int64] = {
+            do {
+                return try container.decode([Int64].self, forKey: .variations)
+            } catch {
+                DDLogError("⛔️ Error parsing `variations` for `Product`: \(error)")
```

Comment:
Would it be useful to pass the productID here so is easier to find out the product that is logging the error? 

```
DDLogError("⛔️ Error parsing `variations` for `Product` \(productID): \(error)")
```

I'm not entirely sure of what data the error will contain as I was unable to replicate this.

Reply:
The error will look like this:

```
typeMismatch(Swift.String, Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "data", intValue: nil), _JSONKey(stringValue: "Index 2", intValue: 2), CodingKeys(stringValue: "images", intValue: nil), _JSONKey(stringValue: "Index 0", intValue: 0), CodingKeys(stringValue: "alt", intValue: nil)], debugDescription: "Expected to decode String but found a number instead.", underlyingError: nil))
```

Also, good point about tracking the product ID! That'll be helpful for troubleshooting. Updated in 2bed57dd321e45b2c8b08fc6485cd21c87497071.